modules/op-release-notes-1-9.adoc

```diff
@@ -0,0 +1,131 @@
+
+* The `chains-secret` and `chains-config` config map are removed after you uninstall the Red Hat OpenShift Pipelines operator. As they contain user data, they should be preserved and not deleted.
+
+// https://issues.redhat.com/browse/SRVKP-2396
+
+[id="fixed-issues-1-9_{context}"]
+== Fixed issues
+
+// .Pipelines
+
+* With this update, after replacing the empty array, the original array will be empty.
+// Vincent Demeester
+
+* Before this update, if duplicate secrets are present in a service account for a pipelines run, it resulted in failure in task pod creation. With this update, this issue is resolved and the task pod is created successfully even if duplicate secrets were present in a service account.
+// Vincent Demeester
+
+* Before this update, by looking at the TaskRun's `spec.StatusMessage` field, users could not distinguish whether the `TaskRun` had been cancelled by the user or by a `PipelineRun` that was part of it. With this update, this issue is resolved and users can distinguish the status of the `TaskRun` by looking at the TaskRun's `spec.StatusMessage` field.
+// Vincent Demeester
+
+* With this update, the user can specify tasks or timeout.
+// Vincent Demeester
+
+* Before this update, a user could not remove webhook validation when deleting old versions of invalid objects. With this update, this issue is resolved and user can remove webhook validation when deleting old versions of invalid objects.
+// Vincent Demeester @vdeemester
+
+* With this update, the user can merge labels and annotations instead of override.
+// Vincent Demeester @vdeemester
+
+// .Triggers
+
+* Before this update, log keys did not have same keys as in pipelines controllers. With this update, this issue has been resolved and the log keys have been updated to match the log stream. The keys in logs have been changed from "ts" to "timestamp", from "level" to "severity", and "message" to "msg".
+// https://issues.redhat.com/browse/SRVKP-1959
+// @KhurramBaig
+
+
+// .CLI
+
+* Before this update, if a pull request was deleted with an unknown status, an error message was not generated. With this update, this issue is resolved and an error message is generated.
+// https://github.com/tektoncd/cli/pull/1684
+// Piyush Garg @piyush-garg
+
+* Before this update, the `kubeconfig` file with bundle commands like `list`, `push` was required to access the cluster. With this update, this issue has been resolved and the `kubeconfig` file is not required to access the cluster.
+// https://github.com/tektoncd/cli/pull/1557
+// Piyush Garg @piyush-garg
+
+* Before this update, if the parent pipelinerun is running while deleting TaskRuns, then TaskRuns will be deleted. With this update, this issue issue is resolved and TaskRuns are not getting deleted if the parent PipelineRun is running.
+// https://github.com/tektoncd/cli/pull/1736
+// Piyush Garg @piyush-garg
+
+* With this command, a checkbox has been for the set bundle max field used in the pipeline controller.
```

Comment:
i dont know what is this

Reply:
Hi @piyush-garg , this text is added with respect to the following: 

Adding a check for the set bundle max used in the pipeline controller

May I know if this text is a part of this overall issue:

Adding a check for the set bundle max used in the pipeline controller
Tkn cli will error if attempting to create a bundle with more objects than allowed by the pipeline controller.

If that is the case, I have added a different entry for the same. It is as follows:

 Before this update, if the user attempted to create a bundle with more objects than the pipeline controller permitted, the Tekton CLI did not display an error message. With this update, this issue is resolved and the Tekton CLI displays an error message message if the user added more objects than the limit permitted in the pipeline.
 
 If you suggest, I could remove this one line entry I added here and only retain the paragraph added above.

Reply:
please remove this

Reply:
@Dhruv-Soni11 please remove this single line entry `With this command, a checkbox has been for the set bundle max field used in the pipeline controller.` and just keep this `Before this update, if the user attempted to create a bundle with more objects than the pipeline controller permitted, the Tekton CLI did not display an error message. With this update, this issue is resolved and the Tekton CLI displays an error message if the user added more objects than the limit permitted in the pipeline.`

Reply:
Got it @rupalibehera. I will remove this line and keep the content as you suggested.